### DISCUSSION TITLE
Do disabled queries still subscribe?

### DISCUSSION THREAD

**Discussion by lockr7** [TanStack/query#5073] [Q&A] [answered]
If I have a disabled query and use `queryClient.setQueryData` with the same query key, will it cause the component to re-render? 

It wasn't clear to me from https://tanstack.com/query/v4/docs/react/guides/disabling-queries, if manually setting query data will cause the query to re-retrieve data.

**Comment by TkDodo** [comment, marked as answer]
Absolutely. The docs say that the query is disabled from automatically running - the observer is still watching the cache entry for changes.
